include/mips/malta.h

```diff
@@ -61,14 +61,21 @@ void platform_init(int argc, char **argv, char **envp, unsigned memsize);
  *   15      Secondary IDE     PIIX4
  */
 
+#define MALTA_PHYS_ADDR_SPACE_BASE 0x00000000
 #define MALTA_PHYS_SDRAM_BASE 0x00000000
 
 #define MALTA_PCI0_MEMORY_BASE 0x10000000
 #define MALTA_PCI0_MEMORY_END 0x17ffffff
 #define MALTA_PCI0_IO_BASE 0x18000000
 #define MALTA_PCI0_IO_END 0x1bdfffff
+#define MALTA_PCI0_TO_ISA_BRIDGE_BASE MALTA_PCI0_IO_BASE
+#define MALTA_PCI0_TO_ISA_BRIDGE_END (MALTA_PCI0_TO_ISA_BRIDGE_BASE + 0x100 - 1)
+#define MALTA_PCI0_EXCLUSIVE_IO_BASE (MALTA_PCI0_IO_BASE + 0x100)
+#define MALTA_PCI0_EXCLUSIVE_IO_END MALTA_PCI0_IO_END
 #define MALTA_CORECTRL_BASE 0x1be00000
+#define MALTA_CORECTRL_END (0x1e000000 - 1)
```

Comment:
Why it's not in computed form, while others are?

Reply:
For no particular reason. I'll replace it with 0x1dffffff.